hivemq-edge/src/main/java/com/hivemq/extensions/core/RestComponentsService.java

```diff
@@ -0,0 +1,17 @@
+package com.hivemq.extensions.core;
+
+import com.hivemq.api.resources.GenericAPIHolder;
+import com.hivemq.extension.sdk.api.annotations.NotNull;
+
+public class RestComponentsService {
```

Comment:
may make sense to add an interface here as it is exposed to the modules

Reply:
yeah ill do so